app/DoctrineMigrations/Version20160812120952.php

```diff
@@ -30,7 +30,20 @@ public function up(Schema $schema)
         $clientsTable = $schema->getTable($this->getTable('oauth2_clients'));
         $this->skipIf($clientsTable->hasColumn('name'), 'It seems that you already played this migration.');
 
-        $clientsTable->addColumn('name', 'blob');
+        if ('sqlite' === $this->connection->getDatabasePlatform()->getName()) {
+            // Can't use $clientsTable->addColumn('name', 'blob');
+            // because of the error:
+            // SQLSTATE[HY000]: General error: 1 Cannot add a NOT NULL column with default value NULL
+            $databaseTablePrefix = $this->container->getParameter('database_table_prefix');
+            $this->addSql('CREATE TEMPORARY TABLE __temp__' . $databaseTablePrefix . 'oauth2_clients AS SELECT id, random_id, redirect_uris, secret, allowed_grant_types FROM ' . $databaseTablePrefix . 'oauth2_clients');
```

Comment:
🤢

Reply:
In fact, I have used `doctrine:migrations:diff` to generate the SQLITE migration, I had to modify it to use the `database_table_prefix`